### PR TITLE
feat(adoption): add starter kit, static build, and experimental streaming SSR

### LINKED ISSUE #48: feat(build): static site generation con minificación por defecto
### What problem does this solve?

Actualmente Axiom no ofrece a los desarrolladores una forma oficial de generar páginas estáticas minificadas. El SSR (`renderToString` en `src/ssr.ts`) produce HTML funcional pero:

1. **El bundle cliente (el `app.js` que acompaña al HTML generado) no se minifica por defecto.** Esto se descubrió al abrir `demo/app.js` asociado a `demo/static.html` y observar el tamaño del archivo — el script del demo pesaba varios KB más de lo necesario porque `Bun.build()` se estaba llamando sin `minify: true`.
2. **El HTML generado por SSR contiene estilos inline por nodo** (`position:absolute;left:0px;top:0px;transform:translate(...)...`) que no se compactan ni deduplican.
3. **El CSS inline opcional (`metadata.inlineStyles`) se emite tal cual**, sin minificar.
4. **No existe una API pública de build estático** en el framework. Los consumidores tienen que orquestar manualmente `renderToString` + `Bun.build` + escritura a disco, replicando la lógica de `demo/build.ts` y `demo/ssr-page.tsx`.

El resultado: cualquier dev que use Axiom para generar sitios estáticos va a servir artefactos más grandes de lo necesario, a menos que sepa configurar el pipeline de bundling por su cuenta.

**Nota:** El fix puntual para el demo (`minify: true` en `demo/build.ts`) se aplica en el PR asociado a esta issue. Esta issue cubre el caso general — que la minificación sea el comportamiento por defecto garantizado para **cualquier usuario del framework** que genere páginas estáticas, no solo el demo.

### Proposed API / behavior

Exponer una API de build estático de primera clase que minifique por defecto. Dos opciones complementarias:

**Opción A — API programática (`buildStatic`)**

```typescript
import { buildStatic } from 'axiom-framework'
import { HomePage } from './pages/home'

await buildStatic({
  routes: [
    { path: '/', component: HomePage },
    { path: '/about', component: AboutPage },
  ],
  outDir: './dist',
  clientEntry: './src/client.ts',
  // Minificación activa por defecto — ship optimizado sin configurar nada.
  minify: true,
  // Opciones opt-out para debugging:
  // minify: false → genera HTML/JS legible
})
```

Responsabilidades:
- Renderizar cada ruta con `renderToString`
- Bundlear el entry cliente con `Bun.build({ minify: true, target: 'browser' })`
- Minificar el HTML resultante (colapsar whitespace entre tags, remover comentarios)
- Minificar `metadata.inlineStyles` si se provee
- Escribir artefactos a disco con nombres estables

**Opción B — CLI wrapper**

```bash
bunx axiom build --routes routes.ts --out dist
# Minify on por defecto; opt-out explícito:
bunx axiom build --no-minify
```

**Contrato clave:** `minify: true` es el default. Los usuarios que quieran artefactos no-minificados (para debugging) deben hacer opt-out explícito. Esto invierte el problema actual donde había que hacer opt-in manual.

### Alternatives considered

1. **Documentar el pipeline manual en el README** — lo consideramos pero falla al objetivo: obliga al usuario a entender Bun.build, SSR, y el orden de operaciones. Contradice la filosofía de "framework, no kit de construcción".
2. **Solo agregar `minify: true` al `demo/build.ts`** — resuelve el demo pero no al usuario final. Es lo que hace el PR actual como fix táctico, pero no es suficiente.
3. **Delegar 100% a Vite/Webpack** — rompe la coherencia: Axiom usa Bun como toolchain de primera clase. Introducir otro bundler duplica dependencias.
4. **Minificar dentro de `renderToString`** — rechazado. `renderToString` debe seguir siendo una función pura de componente → HTML string. La minificación pertenece al pipeline de build, no al runtime de SSR.

### Which part of the system does this affect?

App API (createApp / mount / unmount)

### Constraints check

- [x] The hot path (reflow + commit) remains free of DOM reads — esta feature es 100% build-time, no toca reflow ni commit.
- [x] This does not introduce runtime dependencies — la minificación se hace en build time con las APIs que Bun ya provee (`Bun.build({ minify: true })`). No se agregan deps a `dependencies`.
- [x] This does not break the existing public API — añade nuevas entradas (`buildStatic`, CLI) sin modificar `renderToString`, `createApp`, signals, etc.

### Additional context

**Archivos de referencia para el implementador:**
- `src/ssr.ts` — `renderToString` ya existe y es el input correcto.
- `demo/build.ts` — patrón actual de `Bun.build` con `target: 'browser'`. Ya se agregó `minify: true` en el PR tactical fix; la API pública debe heredar ese default.
- `demo/ssr-page.tsx` — ejemplo de render completo server-side que puede servir de referencia para el contrato de rutas.

**Criterios de aceptación sugeridos:**
1. `buildStatic()` exportada desde el punto de entrada público (`src/index.ts` o subpath `axiom-framework/build`).
2. `minify: true` como default documentado.
3. Test unitario que verifique: dado un componente trivial, el HTML output no contiene comentarios, el JS output del bundle es menor que el no-minificado, y el CSS inline se compacta.
4. Sección en el README con el ejemplo de uso mínimo.
5. El demo se migra a usar `buildStatic` como dogfooding.

### PR DESCRIPTION
**Pull request by naml14**
## Summary

- Add `buildStatic()` as a public static-site-generation API, plus a `build:static` CLI entry point and demo dogfooding for minified static output.
- Add the `create-axiom` starter workflow with project templates, a cookbook, and improved watch-mode rebuild ergonomics for onboarding.
- Add experimental `renderToReadableStream()` and `createServer()` APIs with tests, while refreshing README and release evidence docs to reflect the new adoption surface.

## Type of change

- [ ] `fix` — bug fix (patch bump)
- [x] `feat` — new feature (minor bump)
- [ ] `feat!` / `BREAKING CHANGE` — breaking change (major bump)
- [ ] `perf` — performance improvement (patch bump)
- [ ] `refactor` — no behavior change
- [ ] `test` — tests only
- [ ] `docs` — documentation only
- [ ] `ci` — workflow changes
- [ ] `chore` — maintenance

## Checklist

- [x] Tests added or updated for the changed behavior
- [x] `bun test` passes locally (all tests green)
- [x] `bun run typecheck` passes locally
- [x] This PR links at least one issue (`Closes #...`)
- [x] `docs/V1-0-0-EVIDENCE-LOG.md` was updated (or marked N/A with reason)
- [x] Commit messages follow [Conventional Commits](https://www.conventionalcommits.org/) format
- [x] Hot path invariant respected: no DOM reads added to `src/render/reflow.ts`, `src/render/engines/fast-path.ts`, `src/render/engines/flex.ts`, or `src/render/commit.ts`
- [x] No runtime dependencies added to `src/`

## Related issues

Closes #48
Refs #30

## Testing notes

- `bun test`
- `bun run typecheck`
- `bun run test:coverage`
- `bun test tests/build.test.ts`
- `bun test tests/ssr-stream.test.ts tests/server.test.ts`
- No shell scripts were modified, so shellcheck is N/A.

## Summary by Sourcery

Agregar una API pública para generación de sitios estáticos y una CLI, introducir un andamiaje de proyecto inicial con documentación tipo cookbook, y añadir APIs experimentales de SSR por streaming y de envoltorio de servidor, junto con mejoras de ergonomía para demo/watch y actualizaciones de evidencia de la versión.

New Features:
- Exponer una API estable `buildStatic()` para generación de sitios estáticos con un punto de entrada de CLI `build:static` e integración de snapshots de demo.
- Proporcionar un flujo de trabajo de CLI `create-axiom` con plantillas de proyecto y un cookbook práctico para la incorporación de nuevos usuarios.
- Introducir APIs experimentales `renderToReadableStream()` y `createServer()` para SSR por streaming y un envoltorio de servidor mínimo.

Enhancements:
- Refinar la canalización de compilación de la demo con generación de snapshots estáticos y reconstrucciones en modo watch con debounce usando un registro de logs más claro.
- Actualizar el README y la evidencia/docs para documentar la generación estática, las capacidades de SSR por streaming y el estado de validación de la versión v1.0.0.

Build:
- Añadir scripts para compilaciones de sitios estáticos y andamiaje de proyectos, además de plantillas iniciales conectadas a `axiom-framework`.

Documentation:
- Añadir un cookbook completo con recetas prácticas y actualizar la documentación de compilación estática y la evidencia de la versión v1.0.0.

Tests:
- Añadir pruebas de compilación, SSR por streaming e integración de servidor para validar la salida estática, la corrección del HTML, la inyección de metadatos y el comportamiento básico del servidor.

<details>
<summary>Original summary in English</summary>

## Summary by Sourcery

Add a public static site generation API and CLI, introduce a starter project scaffold with cookbook documentation, and add experimental streaming SSR and server wrapper APIs, along with demo/watch ergonomics and release evidence updates.

New Features:
- Expose a stable buildStatic() API for static site generation with a build:static CLI entry point and demo snapshot integration.
- Provide a create-axiom CLI workflow with project templates and a practical cookbook for onboarding.
- Introduce experimental renderToReadableStream() and createServer() APIs for streaming SSR and a minimal server wrapper.

Enhancements:
- Refine the demo build pipeline with static snapshot generation and debounced watch-mode rebuilds using clearer logging.
- Update README and evidence/docs to document static generation, streaming SSR capabilities, and v1.0.0 validation status.

Build:
- Add scripts for static site builds and project scaffolding, plus starter templates wired for axiom-framework.

Documentation:
- Add a comprehensive cookbook with practical recipes and update static build and v1.0.0 evidence documentation.

Tests:
- Add build, streaming SSR, and server integration tests to validate static output, HTML correctness, metadata injection, and basic server behavior.

</details>

### CODE REVIEW

File: src/build.ts [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+// Internal helpers
+// ============================================================
+
+/** Resolve a route path to a filesystem directory suitable for index.html. */
+function routeToDir(outDir: string, routePath: string): string {
+  // Normalise: strip leading slash, collapse '.' and '..'
+  const clean = posix.normalize(routePath.replace(/^\/+/, '') || '.')
+  if (clean === '.') return outDir
+  return join(outDir, ...clean.split('/'))
+}
```
[the remark this answers is not in the document]
Comment (naml14) [reply src/build.ts:100]: ✅ Resuelto. `routeToDir()` ahora normaliza y valida la ruta, rechaza segmentos `..`/prefijos de drive y además valida contención con `isPathInside(...)` antes de escribir en disco. También quedó cubierto con test (`tests/build.test.ts`: caso `rejects route traversal outside outDir`).

File: src/build.ts [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+// ============================================================
+// Internal helpers
+// ============================================================
+
+/** Resolve a route path to a filesystem directory suitable for index.html. */
+function routeToDir(outDir: string, routePath: string): string {
+  // Normalise: strip leading slash, collapse '.' and '..'
+  const clean = posix.normalize(routePath.replace(/^\/+/, '') || '.')
+  if (clean === '.') return outDir
+  return join(outDir, ...clean.split('/'))
```
[the remark this answers is not in the document]
Comment (naml14) [reply src/build.ts]: ✅ Resuelto. Esta vulnerabilidad de path traversal quedó mitigada en `routeToDir()` con validación explícita de segmentos peligrosos y comprobación de contención en `outDir`. Además quedó protegida por test de regresión en `tests/build.test.ts`.

File: src/server.ts [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+  return bun
+}
+
+// ============================================================
+// createServer
+// ============================================================
+
+/**
+ * Minimal server that integrates Axiom SSR + static file serving.
+ * Wraps Bun.serve().
```
[the remark this answers is not in the document]
Comment (naml14) [reply src/server.ts:83]: ✅ Resuelto. El servidor ya no concatena rutas; ahora usa `resolveStaticFilePath(...)` con `decodeURIComponent`, normalización/resolución y verificación `isPathInside(...)` para impedir escapes de `staticDir`. También hay test de traversal en `tests/server.test.ts`.

File: src/build.ts [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+//     │   ├─ renderToString(component, { metadata }) → HTML string
+//     │   └─ writeFile(outDir/<path>/index.html)
+//     │
+//     ├─ Bun.build({ entrypoints, outDir/assets, minify }) → JS bundle(s)
+//     │
+//     └─ write asset-manifest.json
+// ============================================================
+
+import { mkdir, writeFile } from 'node:fs/promises'
+import { join, posix } from 'node:path'
```
[the remark this answers is not in the document]
Comment (naml14) [reply src/build.ts]: ✅ Ya está cubierto. `src/build.ts` importa `relative` (junto con `isAbsolute`, `resolve`, etc.) y se usa en `isPathInside(...)` para validar contención de rutas.

File: src/server.ts [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+            const file = Bun.file(filePath)
+            if (await file.exists()) {
+              return new Response(file)
+            }
+          }
+
+          // Try route match
+          for (const route of routes) {
+            if (matchRoute(url.pathname, route.path)) {
+              const stream = renderToReadableStream(route.component, ssr)
```
[the remark this answers is not in the document]
Comment (naml14) [reply src/server.ts]: ✅ Resuelto. `createServer()` ya pasa metadata por ruta al stream con `metadata: route.metadata ?? ssr?.metadata`.

File: src/server.ts [hunk 6]
```diff
@@ -0,0 +1,102 @@
+// ============================================================
+// axiom-framework — Server Wrapper
+// ============================================================
+
+import type { ComponentDefinition } from './core/types.js'
+import type { StreamSSROptions } from './ssr-stream.js'
+import { renderToReadableStream } from './ssr-stream.js'
```
[the remark this answers is not in the document]
Comment (naml14) [reply src/server.ts:8]: ✅ Resuelto. En `src/server.ts` ya están los imports necesarios y el soporte de metadata por ruta (`SSRMetadata`) para alinear `createServer` con `buildStatic`.

File: src/server.ts [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+    get port() { return actualPort },
+    serve() {
+      server = Bun.serve({
+        port: configuredPort,
+        async fetch(req) {
+          const url = new URL(req.url)
+
+          // Try static file first
+          if (staticDir) {
+            const filePath = staticDir + url.pathname
```
[the remark this answers is not in the document]
Comment (naml14) [reply src/server.ts]: ✅ Resuelto por una solución más robusta que `join` simple: `resolveStaticFilePath(...)` con decode + normalize/resolve + `isPathInside(...)` para prevenir traversal y comportamientos inconsistentes.

File: scripts/create-axiom.ts [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+  for (const [src, dest] of templates) {
+    const content = await readFile(join(TEMPLATES_DIR, src), 'utf8')
+    // Replace placeholder project name in package.json
+    const final = src === 'package.json'
+      ? content.replace('"my-axiom-app"', JSON.stringify(projectName))
+      : content
+    await writeFile(join(projectDir, dest), final, 'utf8')
+    console.log(`  Created ${dest}`)
+  }
```
[the remark this answers is not in the document]
Comment (naml14) [reply scripts/create-axiom.ts:51]: ✅ Resuelto. `scripts/create-axiom.ts` ya parsea `package.json` como JSON, actualiza `name` y vuelve a serializar (evitando reemplazos frágiles por string).

File: src/server.ts [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+
+// ============================================================
+// Types
+// ============================================================
+
+export interface AxiomServerOptions {
+  routes: Array<{
+    path: string
+    component: ComponentDefinition<void>
+  }>
```
[the remark this answers is not in the document]
Comment (naml14) [reply src/server.ts:19]: ✅ Resuelto. `AxiomServerOptions.routes` ya acepta `metadata?: SSRMetadata` por ruta, y esa metadata se reenvía al renderer de streaming.

File: .atl/changes/streaming-ssr/proposal.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+| `src/index.ts` | Modify | Export streaming + server |
+| `tests/ssr-stream.test.ts` | **Create** | Streaming tests |
+| `tests/server.test.ts` | **Create** | Server integration tests |
+
+## Risks
+
+| Risk | Likelihood | Mitigation |
+|------|------------|------------|
+| Streaming SSR es complejo de implementar correctamente | High | Marcar como @experimental en v1.x; release estable en v1.1+ |
+| Incompatibilidad con hydration actual | Medium | Streaming produce HTML diferente (con templates). Separar contractos |
```
[the remark this answers is not in the document]
Comment (naml14) [reply .atl/changes/streaming-ssr/proposal.md]: ✅ Resuelto. El typo quedó corregido en `.atl/changes/streaming-ssr/proposal.md`: `contractos` → `contratos`.

File: src/build.ts [hunk 11]
```diff
@@ -0,0 +1,173 @@
...
+      entrypoints: assets.entrypoints,
+      outdir: assetsOutDir,
+      target: 'browser',
+      minify,
+    })
+
+    if (!result.success) {
+      const msgs = result.logs.map(l => String(l)).join('\n')
+      throw new Error(`buildStatic: JS bundle failed.\n${msgs}`)
+    }
```
[the remark this answers is not in the document]
Comment (naml14) [reply src/build.ts:181]: ✅ Resuelto por alineación de contrato/documentación. Mantuvimos `minify` para el bundle JS y actualizamos docs para dejar explícito que la minificación HTML/CSS aún no está implementada (`docs/STATIC-BUILD-MINIFICATION.md` y secciones de README).

File: src/ssr-stream.ts [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+ * then replaced when the Promise resolves.
+ *
+ * @experimental — API may change in minor versions
+ */
+export function renderToReadableStream(
+  component: ComponentDefinition<void>,
+  options?: StreamSSROptions
+): ReadableStream<Uint8Array> {
+  // For the initial implementation, we wrap renderToString in a stream.
+  // Async boundary support will be added in a follow-up.
```
[the remark this answers is not in the document]
Comment (naml14) [reply src/ssr-stream.ts]: ✅ Resuelto. `src/ssr-stream.ts` ahora documenta el comportamiento real (single-chunk stream), elimina la semántica no implementada de boundaries en `StreamSSROptions` y deja claro el estado `@experimental`.

File: tests/build.test.ts [hunk 13]
```diff
@@ -0,0 +1,211 @@
...
+})
+
+// ============================================================
+// Tests
+// ============================================================
+
+describe('buildStatic()', () => {
+  // ---- Filesystem output ----
+
+  test('2.1 writes files to outDir', async () => {
```
[the remark this answers is not in the document]
Comment (naml14) [reply tests/build.test.ts:93]: ✅ Resuelto. Se añadió cobertura de integración para la rama `assets` (`tests/build.test.ts`, caso `2.0 bundles assets with Bun.build defaults`) verificando `entrypoints`, `outdir` por defecto (`<outDir>/assets`) y `minify: true`, además de presencia en `result.files` y manifiesto.

File: docs/V1-0-0-EVIDENCE-LOG.md [hunk 14]
```diff
@@ -36,6 +36,40 @@ Plantilla:
...
+- Issue: #30 (épica 1.0.0)
+- PR: TBD
+- Verificación:
+  - [x] `bun run validate:api` — 121 stable exports, 0 beta/experimental/untagged
+  - [x] `bun run test:coverage` — 96.95% line coverage >= threshold 85%
+  - [x] `bun test tests/ssr.test.ts tests/hydration.test.ts` — 20 pass / 0 fail
+  - [x] `bun test` — 540 pass / 2 skip / 0 fail
+  - [x] `bun run typecheck` — 0 errors
+- Evidencia:
+  - API stability: `docs/STABILITY.md` exists with contract v1.0.0
```
[the remark this answers is not in the document]
Comment (naml14) [reply docs/V1-0-0-EVIDENCE-LOG.md]: ✅ Resuelto. Se corrigió la evidencia de estabilidad/API para reflejar correctamente exports experimentales y sus conteos en `docs/V1-0-0-EVIDENCE-LOG.md` + `docs/STABILITY.md`.

File: tests/build.test.ts [hunk 15]
```diff
@@ -0,0 +1,211 @@
...
+    const aboutHtml = await readFile(join(outDir, 'about', 'index.html'), 'utf8')
+    expect(aboutHtml).toContain('About')
+
+    const contactHtml = await readFile(join(outDir, 'contact', 'index.html'), 'utf8')
+    expect(contactHtml).toContain('Contact')
+  })
+
+  // ---- Edge cases ----
+
+  test('2.5 empty routes array produces only the manifest', async () => {
```
[the remark this answers is not in the document]
Comment (naml14) [reply tests/build.test.ts:221]: ✅ Resuelto. Se añadió test de fallo de bundling (`tests/build.test.ts`, caso `2.5 surfaces Bun.build logs on asset bundling failure`) que comprueba rechazo con mensaje incluyendo logs concatenados.

File: README.md [hunk 16]
```diff
@@ -36,9 +36,17 @@ bun add axiom-framework
...
 ```
 
----
+### Quick Start (CLI)
+
+```bash
+bun run create-axiom my-app
+cd my-app
+bun dev
+```
```
[the remark this answers is not in the document]
Comment (naml14) [reply README.md:47]: ✅ Resuelto. Ya se publicó el CLI como parte del paquete: `package.json` incluye `bin.create-axiom` y se añadieron `scripts/create-axiom.ts` + `scripts/templates` en `files`, por lo que el comando es accesible para consumidores.

File: src/index.ts [hunk 17]
```diff
@@ -112,6 +112,18 @@ export type {
   GridProps,
 } from './syntax/index.js'
 
+// --- Streaming SSR (experimental) ---
+export { renderToReadableStream } from './ssr-stream.js'
+export type { StreamSSROptions } from './ssr-stream.js'
+
+// --- Server (experimental) ---
+export { createServer } from './server.js'
```
[the remark this answers is not in the document]
Comment (naml14) [reply src/index.ts:123]: ✅ Resuelto. `src/index.ts` ya marca explícitamente como `@experimental` los exports de streaming SSR y server (`renderToReadableStream`, `StreamSSROptions`, `createServer`, `AxiomServer`, `AxiomServerOptions`).

File: src/build.ts [hunk 18]
```diff
@@ -0,0 +1,173 @@
...
+ * @stable
+ */
+export async function buildStatic(options: BuildStaticOptions): Promise<BuildResult> {
+  const start = Date.now()
+  const {
+    routes,
+    outDir,
+    minify = true,
+    assets,
+  } = options
```
[the remark this answers is not in the document]
Comment (naml14) [reply src/build.ts:146]: ✅ Resuelto. Se añadió prueba específica del contrato `minify` por defecto en la rama de `assets` (`tests/build.test.ts`, caso `2.0 ...`) inspeccionando las opciones pasadas a `Bun.build` y verificando `minify: true`.

File: docs/COOKBOOK.md [hunk 19]
```diff
@@ -0,0 +1,416 @@
...
+  stack({ gap: 16, padding: 32 },
+    h('h1', null, 'Axiom SSR'),
+    h('p', null, 'Esta página fue renderizada en el servidor.'),
+    h('button', { id: 'hydrate-btn' }, 'Clic para interacción'),
+  )
+)
+
+// renderToString devuelve HTML + metadata de hidratación
+const { html, metadata } = await renderToString(HomePage, {
+  meta: { title: 'Axiom SSR Demo', description: 'Server-side rendering con Axiom' },
```
[the remark this answers is not in the document]
Comment (naml14) [reply docs/COOKBOOK.md]: ✅ Resuelto. La receta SSR del cookbook ya usa el contrato actual: `renderToString()` retorna HTML string y la metadata se pasa por `options.metadata`.

File: src/build.ts [hunk 20]
```diff
@@ -0,0 +1,173 @@
...
+    /** JS entry points to bundle for the browser. */
+    entrypoints: string[]
+    /** Output directory for bundled assets. Default: outDir/assets */
+    outDir?: string
+  }
+}
+
+export interface BuildResult {
+  routes: number
+  /** Absolute paths of every file written to disk. */
```
[the remark this answers is not in the document]
Comment (naml14) [reply src/build.ts:51]: ✅ Resuelto. `BuildResult.files` ahora es coherente con su contrato: se normalizan rutas a absolutas (`resolve(...)` para `outDir`, outputs de Bun y manifiesto), y el manifiesto persiste esas rutas normalizadas.

File: src/build.ts [hunk 21]
```diff
@@ -0,0 +1,173 @@
...
+  path: string
+  component: ComponentDefinition<void>
+  metadata?: SSRMetadata
+}
+
+export interface BuildStaticOptions {
+  routes: StaticRoute[]
+  outDir: string
+  /** URL prefix for all routes. Default: '/' */
+  basePath?: string
```
[the remark this answers is not in the document]
Comment (naml14) [reply src/build.ts]: ✅ Resuelto. `basePath` se retiró de `BuildStaticOptions` para evitar una opción pública no operativa; el contrato ya no sobrepromete ese comportamiento.

File: scripts/build-static.ts [hunk 22]
```diff
@@ -0,0 +1,123 @@
...
+ *
+ * This script looks for a `src/static-routes.ts` file to define pages,
+ * or falls back to a single-page build from `src/app.ts`.
+ *
+ * For advanced usage, import `buildStatic()` directly from 'axiom-framework'.
+ */
+
+import { existsSync } from 'node:fs'
+import { join, dirname } from 'node:path'
+import { fileURLToPath } from 'node:url'
```
[the remark this answers is not in the document]
Comment (naml14) [reply scripts/build-static.ts]: ✅ Resuelto. Se eliminaron imports no usados en `scripts/build-static.ts`; el archivo quedó con imports mínimos (`existsSync` y `join`).

File: README.md [hunk 23]
```diff
@@ -194,19 +202,27 @@ Important detail: `keyBy` belongs to `For`. JSX `key` is accepted by the runtime
 ┌─────────────────────────────────────────────────────────────┐
 │                     PUBLIC API                               │
 │  signal()  computed()  effect()  defineComponent()  createApp() │
+│  buildStatic()  renderToReadableStream()  createServer()       │
+│  renderToString()  createRouter()  createAxiom CLI            │
```
[the remark this answers is not in the document]
Comment (naml14) [reply README.md]: ✅ Resuelto. El diagrama de README ya usa el nombre correcto `create-axiom` (alineado con scripts/CLI actual).

File: scripts/templates/package.json [hunk 24]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "my-axiom-app",
+  "version": "0.1.0",
+  "type": "module",
+  "scripts": {
+    "dev": "bun run demo/server.ts --watch",
```
[the remark this answers is not in the document]
Comment (naml14) [reply scripts/templates/package.json]: ✅ Resuelto. La plantilla ahora incluye `dev-server.ts` y el script `dev` apunta a `bun --watch run dev-server.ts`, por lo que `bun dev` funciona en proyectos recién scaffolded.
